.travis.yml

```diff
@@ -5,15 +5,10 @@ sudo: false
 dart:
   - stable
 
-dart_task:
-  - test: --platform vm
```

Comment:
Not familiar with Travis's Dart support, what's the impact of removing this?

Reply:
So this was a little weird:

We used this task to run the tests, but then we ran presubmit prior that would execute `pub run test`. Instead, I've changed this to run the formatting check, then I run the tests afterwords. I think we could potentially remove the presubmit.dart script after this.

I think we could avoid specifying the platform since we're just running everything within the Dart VM.